style.css

```diff
@@ -0,0 +1,503 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* GLOBAL SETTINGS */
+
+.flex-horizontal {
+  display: flex;
+}
+
+.flex-vertical {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
To give an idea of how to keep the projects CSS DRY. This could be refactor to:
```
.flex-horizontal,
.flex-vertical  {
  display: flex;
}

.flex-vertical {
  flex-direction: column;
}
```
Or even better create a utility selector:
```
.d-flex {
  display: flex;
}

.flex-vertical {
  flex-direction: column;
}
```